### PR TITLE
650 ascii doc

### LINKED ISSUE #650: Flank ASCII doc
See if the new asciidoc support in picocli can be used to generate complete documentation for the Flank CLI.

- https://github.com/remkop/picocli/releases/tag/v4.2.0

### PR DESCRIPTION
**Pull request by jan-gogo**
Fixes #650 

## Checklist

- [x] release_notes.md updated

### CODE REVIEW

File: test_runner/build.gradle.kts [hunk 1]
```diff
@@ -237,3 +236,34 @@ tasks.create("updateFlank", Exec::class.java) {
...
+    setBackends("html5")
+    sourceDir = file("${project.buildDir}/generated-picocli-docs")
+    outputDir = file("${project.rootDir}/docs")
+    logDocuments = true
+}
+
+tasks.assemble {
+    dependsOn(tasks.asciidoctor)
+}
+// end --- ASCII doc generation ---
```
Comment (bootstraponline) [inline test_runner/build.gradle.kts:269]: image
let's end with a newline :)

**Comment by bootstraponline** [comment]
The html files look great. We probably want to figure out a way to publish them. I had two ideas. One is githack.com and the other is github pages.

We could an index page and then linking that from the readme using http://raw.githack.com/ so that it's easy to view the HTML files.

https://rawcdn.githack.com/Flank/flank/51c89128921c8ff39c4b165acc496ef606d1a6d3/test_runner/docs/html5/flank.jar_-android-doctor.html

Another idea is to use github pages and push the html files to a `gh-pages` branch
https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

**Comment by bootstraponline** [comment]
Another idea is if GitHub.com natively renders asciidoc then maybe that’s a better in repo format than HTML.

**Comment by jan-gogo** [comment]
Ok, I'll check these options

File: test_runner/build.gradle.kts (hunk 1 above)
Comment (jan-gogo) [reply test_runner/build.gradle.kts:269]: Ahh sorry. No more missing newlines, finally I checked `Ensure line feed at file end on save` in IDE.

**Comment by jan-gogo** [comment]
> If GitHub.com natively renders asciidoc then maybe that’s a better in repo format than HTML.

ASCIIdoc looks nice and works out of the box. Good effect without effort. 👍

**Comment by bootstraponline** [comment]
looks excellent! ⭐

**Pull request merged**